var/spack/repos/builtin/packages/py-openmc/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOpenmc(PythonPackage):
+    """The OpenMC project aims to provide a fully-featured Monte Carlo particle
+       transport code based on modern methods. It is a constructive solid
+       geometry, continuous-energy transport code that uses ACE format cross
+       sections. The project started under the Computational Reactor Physics
+       Group at MIT."""
+
+    homepage = "http://openmc.readthedocs.io/"
+    url = "https://github.com/openmc-dev/openmc/tarball/v0.10.0"
```

Comment:
```suggestion
    url = "https://github.com/openmc-dev/openmc/tarball/v0.12.0"
```

Reply:
I will leave it at 0.10.0 because we aren't fetching 0.12.0 from URL.

Reply:
In that case:
```suggestion
    url = "https://github.com/openmc-dev/openmc/tarball/v0.11.0"
```
We want the version to match the url for `spack url summary` to work